packages/assets-controllers/src/NftController.ts

```diff
@@ -644,6 +644,16 @@ export class NftController extends BaseController<
         this.#useIpfsSubdomains,
       );
     }
+    if (tokenURI.startsWith('data:image/svg+xml;base64')) {
```

Comment:
This fix seems fine, but i'll note that `tokenURI` is meant to point to JSON, not directly to the image:

```

    /// @notice A distinct Uniform Resource Identifier (URI) for a given asset.
    /// @dev Throws if `_tokenId` is not a valid NFT. URIs are defined in RFC
    ///  3986. The URI may point to a JSON file that conforms to the "ERC721
    ///  Metadata JSON Schema".
    function tokenURI(uint256 _tokenId) external view returns (string);
```

But we can do reasonable things to support non compliant NFTs.

There could theoretically be formats other than SVG in the data URI, like png/gif.  If we wanted to check `tokenURI.startsWith('data:image/` instead.  Not sure if the wallets would render them correctly.

Reply:
That makes sense, ill relax the check